content/reports/1000_IBM Power - Virtual Machines/011_All PowerHMC VMs.yaml

```diff
@@ -0,0 +1,86 @@
+---
+  title: IBM PowerHMC VMs
+  rpt_group: Custom
+  rpt_type: Custom
+  priority:
+  db: ManageIQ::Providers::InfraManager::Vm
+  cols:
+  - name
+  - os_image_name
+  - ram_size
+  - entitled_processors
+  - processor_pin_policy
+  - num_cpu
+  - num_disks
+  - disk_size
+  - ipaddresses
```

Comment:
I think we're only collecting IP addresses for hosts in inventory, not VMs. Is that correct? If so, let's remove `ipaddresses` so we dont' have an empty column.